docs/contributing/adding_a_feature.md

```diff
@@ -87,6 +87,8 @@ Then we conditionally move cilium-operator to masters:
       {{ '{{ end }}' }}
 ```
 
+After changing manifest files remember to run `bash hack/update-expected.sh` in order to get updated [manifestHash](https://github.com/kubernetes/kops/blob/master/upup/pkg/fi/cloudup/tests/bootstrapchannelbuilder/cilium/manifest.yaml#L74) values.
```

Comment:
Thanks for making this bit more clear. Note that not all manifests have tests that requires `update-expected`.

Perhaps split this to a dedicated commit?

Reply:
done